src/global-table.ts

```diff
@@ -1,22 +1,103 @@
-import { IResource, RemovalPolicy, Resource, Stack } from 'aws-cdk-lib';
+import { IResource, RemovalPolicy, Resource, Stack, Token } from 'aws-cdk-lib';
 import * as ddb from 'aws-cdk-lib/aws-dynamodb';
+import * as iam from 'aws-cdk-lib/aws-iam';
 import { Construct } from 'constructs';
+import * as perms from './perms';
 
 export interface IGlobalTable extends IResource {
+
+  /**
+   * Adds an IAM policy statement associated with this table to an IAM
+   * principal's policy.
+   * @param grantee The principal (no-op if undefined)
+   * @param actions The set of actions to allow (i.e. "dynamodb:PutItem", "dynamodb:GetItem", ...)
+   */
+  grant(grantee: iam.IGrantable, ...actions: string[]): iam.Grant;
+
+  /**
+   * Permits an IAM principal all data read operations from this table:
+   * BatchGetItem, Query, GetItem, Scan, ConditionCheckItem, DescribeTable.
+   *
+   * Appropriate grants will also be added to the customer-managed KMS key
+   * if one was configured.
```

Comment:
Remove this from the comment for now, since there is no way for customers to configure a KMS key yet, it is confusing.  (Same comment applies for the other grant methods)